tests/unit/query_plan_match_filter_return.cpp

```diff
@@ -1006,6 +1651,8 @@ class QueryPlanExpandWeightedShortestPath : public testing::Test {
     for (int i = 0; i < 5; i++) {
       v.push_back(dba.InsertVertex());
       ASSERT_TRUE(v.back().SetProperty(prop.second, memgraph::storage::PropertyValue(i)).HasValue());
+      auto label = "l" + std::to_string(i);
```

Comment:
use += instead of +

Reply:
so code should look like
```auto label = "l";
   label+=std::to_string(i);``` ?

Reply:
Or use some sort of string formatting, such as stringstream